src/lib/curl/all.mk

```diff
@@ -6,7 +6,7 @@ endif
 
 SOURCES		:= base.c io.c
 
-SRC_CFLAGS	:= 
-TGT_LDLIBS	:= -lcurl
+SRC_CFLAGS	:= -isystem /usr/local/Cellar/curl/7.69.1/include
```

Comment:
the file `src/lib/curl/all.mk` is generated by `configure`, and shouldn't be committed.